moto/iam/responses.py

```diff
@@ -655,6 +673,34 @@ def untag_role(self):
         return template.render()
 
 
+LIST_ENTITIES_FOR_POLICY_TEMPLATE = """<ListEntitiesForPolicyResponse>
+ <ListEntitiesForPolicyResult>
+ <PolicyRoles>
```

Comment:
Please update this template so that it returns a dynamic list consistent with the AWS API.

Reply:
I has been updated to return dynamic data.